src/ConnectedMode.UnitTests/Suppressions/IssueServerEventsListenerTests.cs

```diff
@@ -0,0 +1,103 @@
+﻿/*
```

Comment:
These test were copied from TainServerEventsListenerTests and adjusted to match the new skeleton.